cmake/SSGCommon.cmake

```diff
@@ -129,11 +129,12 @@ macro(ssg_build_shorthand_xml PRODUCT)
     )
 endmacro()
 
-macro(ssg_build_xccdf_unlinked PRODUCT)
+macro(ssg_build_xccdf_unlinked PRODUCT DISA_STIG_VERSION)
```

Comment:
I am not a big fan of this. Couldn't you run it as a separate command only for products that have the refs?

Reply:
If we want to add the references only to the unlinked file and rely on the build system to propagate it, I don't see other option. However, if we add the references to the final xccdfs and DSs we could make it more customized.

What do you think?

Reply:
@WesleyCeraso Why do we need these references to be added to build intermediary files? What does this propagation from build system add to these references?

Reply:
@yuumasato If we add the references at the beginning of the build process, they will be copied to the final xccdf, xccdf-1.2, data streams and any other output that carries the rules.
On the other hand, if we add the references after the build process we'll have to add them for each output file.

Reply:
What are the output files? Are there others besides XCCDF and DataStream files?